src/components/TaskCard/index.jsx

```diff
@@ -1,31 +1,61 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import PropTypes from "prop-types";
 import { useDispatch } from "react-redux";
-import { deleteTask } from "store/actions";
-import { DELETE, DELETE_ALT } from "utils/constant/images";
+import { deleteTask, completeTask } from "store/actions";
+import {
+  COMPLETE,
+  COMPLETE_ALT,
+  DELETE,
+  DELETE_ALT,
+} from "utils/constant/images";
+import { getDateDifference } from "utils/helpers/getDateDifference";
 import style from "components/TaskCard/index.module.scss";
 import { checkDateString } from "utils/helpers/propCustomValidation";
 import { dateFormatter } from "utils/helpers/dateFormatter";
+import classNames from "classnames";
+
+const TaskCard = ({ id, task, createdTime, completed }) => {
+  const [completedDate, setCompletedDate] = useState(null);
+  useEffect(() => {
+    setCompletedDate(getDateDifference(createdTime));
+  }, []);
 
-const TaskCard = ({ id, task, createdTime }) => {
   const dispatch = useDispatch();
+
   const deleteAction = () => {
     dispatch(deleteTask(id));
   };
 
+  const completeAction = () => {
+    dispatch(completeTask(id));
+  };
+
   return (
     <div className={style.container}>
-      <p>{task}</p>
-      <p>{`Created at: ${dateFormatter(createdTime)}`}</p>
+      <h1 className={classNames(style.task, { [style.taskDone]: completed })}>
+        {task}
+      </h1>
+      <p className={style.dateText}>{`Created at: ${dateFormatter(
+        createdTime
+      )}`}</p>
```

Comment:
```suggestion
      <p className={style.dateText}>Created at: {dateFormatter(createdTime)}</p>
```
Following the style of line 49. Otherwise current one is also fine.